mlflow/server/js/src/MlflowRouter.tsx

```diff
@@ -38,7 +38,7 @@ export const MlflowRouter = ({
     [],
   );
   return (
-    <>
+    <div css={{ display: 'flex', flexDirection: 'column', minHeight: '100vh' }}>
```

Comment:
@hubertzub-db 
edited `MlflowRouter` as well! i just copied the CSS from the `App.tsx` div